ui/src/App.vue

```diff
@@ -25,24 +27,39 @@ export default {
   },
   data() {
     return {
-      isRemote: {},
-      employmentTypes: {},
-      locations: {},
-      skills: {}
+      isRemote: [],
+      employmentTypes: [],
+      locations: [],
+      skills: []
     }
   },
-  methods: {
-    onSubmit() {
-      axios.get(mockAPI)
+  mounted() {
+    axios.get(mockAPI)
```

Comment:
Eventually we should separate out API callers to a separate place.. but fine for now.

Reply:
Removed onSubmit method for now.